Resources/views/Collector/migrations.html.twig

```diff
@@ -0,0 +1,192 @@
+{% extends app.request.isXmlHttpRequest ? '@WebProfiler/Profiler/ajax_layout.html.twig' : '@WebProfiler/Profiler/layout.html.twig' %}
```

Comment:
not necessary, as this panel does not perform any hack ajax-based loading like the DoctrineBundle one.